ngi_pipeline/tests/test_workflows.py

```diff
@@ -6,7 +6,7 @@
 import tempfile
 import unittest
 
-from ngi_pipeline.conductor.flowcell import process_demultiplexed_flowcells
+#from ngi_pipeline.conductor.flowcell import process_demultiplexed_flowcells
```

Comment:
Ah, mainly because it looked like this was a main part of the actual test, so removing them will make the test a bit useless I think. I thought I'd have a closer look at it when I start writing tests 🙂

Reply:
Ah ok 👍 Sounds good